util/prefix_validate_test.go

```diff
@@ -0,0 +1,40 @@
+package util_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/thenativeweb/get-next-version/util"
+)
+
+func TestIsValidTagPrefix(t *testing.T) {
+	for _, tt := range []struct {
```

Comment:
Can we find a better (i.e. more descriptive name for `tt`)?

(Actually, right now, I couldn't say what `tt` stands for. What about e.g. `testcase`?)